Supertext.Base/Common/IWildcardChecker.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Supertext.Base.Common
+{
+    public interface IWildcardChecker
```

Comment:
Instead of the name `WildcardChecker` why not `RegexChecker`? You've proved in your first test that there's no need for a wildcard (*) but that your implementation is using regex to determine the viability of the value, so why name it `WildcardChecker` if a wildcard is not necessarily required?

Reply:
Because it's about the wildcard. Regex is internal details to implement the requirements at the moment. I could have used some other method. And *.xml for example is not valid regex. Or some*ing has a different meaning in regex.